.github/CODEOWNERS

```diff
@@ -0,0 +1,7 @@
+# To add in a CODEOWNER, please add the following under the Global Rule line (remove the #):
+# *           @your-username
+
+# To add in multiple CODEOWNERS, please follow the following format:
+# *           @username-1 @username-2
+
+# Global rule:
```

Comment:
Great! I'd like to minimize the amount of thinking folks have to do and also provide a bit of context and a gentle reminder as to what they're signing up for when they submit a sample.

What do you think of the below? I'd like to remove the commented out `*` entirely and provide a non-viable handle (ie, something that uses a @~ maybe), but I'm unsure if it will error out when a PR is opened. I guess we could test it out.

```suggestion
# Code owners are the default owners for everything in
# this repository. The owners listed below will be requested for
# review when a pull request is opened.
#*       @global-owner1 @global-owner2
```

Reply:
this sounds good to me! updating it!